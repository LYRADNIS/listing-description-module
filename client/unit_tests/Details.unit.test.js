import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import TheSpace from '../src/TheSpace.jsx';
import InteractionWithGuest from '../src/InteractionWithGuests.jsx';

it('shallow renders theSpace correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<TheSpace ltr={"left-to-right"}/>);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});

it('shallow renders InteractionWithGuest correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<InteractionWithGuest ltr={"testing IWG"}/>);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});

// it('shallow renders guestAccess correctly', () => {
//   const renderer = new ShallowRenderer();
//   renderer.render(<guestAccess ltr={"testing IWG"}/>);
//   const tree = renderer.getRenderOutput();
//   expect(tree).toMatchSnapshot();
// });

// it('shallow renders otherThingsToNote correctly', () => {
//   const renderer = new ShallowRenderer();
//   renderer.render(<otherThingsToNote ltr={"testing IWG"}/>);
//   const tree = renderer.getRenderOutput();
//   expect(tree).toMatchSnapshot();
// });
