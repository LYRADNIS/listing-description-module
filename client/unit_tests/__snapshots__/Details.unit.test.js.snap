// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow renders theSpace correctly 1`] = `
<div
  dir="ltr"
>
  <styled.p>
    <styled.span>
      <styled.span>
        left-to-right
      </styled.span>
    </styled.span>
  </styled.p>
</div>
`;
